modules/apps/27-interchain-accounts/host/client/cli/cli.go

```diff
@@ -20,3 +21,20 @@ func GetQueryCmd() *cobra.Command {
 
 	return queryCmd
 }
+
+// NewTxCmd creates and returns the tx command
+func NewTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        "host",
+		Short:                      "interchain-accounts host subcommands",
```

Comment:
```suggestion
		Short:                      "IBC interchain accounts host transaction subcommands",
```

See [this PR](https://github.com/cosmos/ibc-go/pull/2382).